packages/client/lib/net/peer/rlpxpeer.ts

```diff
@@ -10,8 +10,8 @@ import { Protocol, RlpxSender } from '../protocol'
 import { Peer, PeerOptions } from './peer'
 
 const devp2pCapabilities: any = {
-  eth62: Devp2pETH.eth62,
```

Comment:
I am assuming here that Eth/62 is removed here since Eth/64 changes Eth/62 (Status message), while Eth/63 introduces new methods on top of Eth/62?

Reply:
Eth/62 is reaching EOL and geth will remove support soon.